client/main.lua

```diff
@@ -246,3 +263,83 @@ function DrawText3D(x, y, z, text)
     DrawRect(0.0, 0.0+0.0125, 0.017+ factor, 0.03, 0, 0, 0, 75)
     ClearDrawOrigin()
 end
+
+function JailIntro()
+    local Ped = PlayerPedId()
+    DoScreenFadeOut(10)
+    FreezeEntityPosition(Ped, true)
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'handcuff', 1.0)   
+    SetPedComponentVariation(Ped, 1, -1, -1, -1)
+    ClearPedProp(Ped, 0)
+    Citizen.Wait(1000)
+    SetEntityCoords(Ped,Config.Locations["takephotos"].coords.x, Config.Locations["takephotos"].coords.y, Config.Locations["takephotos"].coords.z)
+    SetEntityHeading(Ped,270.0)
+    Citizen.Wait(1500) 
+    DoScreenFadeIn(500)
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000) 
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000)     
+    SetEntityHeading(Ped,-355.74) 
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000)  
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000)         
+    SetEntityHeading(Ped,170.74) 
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000) 
+     TriggerEvent('InteractSound_CL:PlayOnOne', 'photo', 0.4)
+    Citizen.Wait(3000)       
+    SetEntityHeading(Ped,270.0)
+    Citizen.Wait(2000)
+    DoScreenFadeOut(1100)   
+    Citizen.Wait(2000)
+    TriggerEvent('InteractSound_CL:PlayOnOne', 'jaildoor', 1.0)
+end
+
+function JailIclothes()
+	local ped = ped
+	local model = GetEntityModel(ped)
+	local clothesrand = math.random(1,2)
+    if model == `mp_f_freemode_01` then
+		if clothesrand == 1 then
+			ClearPedProp(ped, 0)
```

Comment:
Instead of `ClearPedProp(ped, 0)` everywhere, you should add it after the model check.

Reply:
I did all the code [here](https://github.com/qbcore-framework/qb-prison/pull/7#discussion_r708647219)